salt/orchestrate/edx/backup.sls

```diff
@@ -61,13 +61,23 @@ deploy_backup_instance_to_{{ ENVIRONMENT }}:
                 - {{ salt.boto_secgroup.get_group_id(
                      'consul-agent-{}'.format(ENVIRONMENT), vpc_name=VPC_NAME) }}
           block_device_mappings:
-            - DeviceName: /dev/xvda
+            - DeviceName: /dev/xvdb
```

Comment:
When we specify the block device mappings it's necessary to list both drives, otherwise it will still only create one.